config/environments/test.rb

```diff
@@ -52,5 +52,5 @@
   config.action_controller.raise_on_missing_callback_actions = true
 
   # Use inline adapter for Active Job in tests for faster execution
-  config.active_job.queue_adapter = :inline
+  config.active_job.queue_adapter = :test
```

Comment:
Flagging this change (to enable use of `assert_enqeued`) as it impacts all test files.

Reply:
awesome, sounds good to me